feat(vps): add question to faq (#4525)
scaleway/docs-content: faq/dedibox.mdx

The speed of your connection will return to normal at the start of the next billing cycle when your data allocation resets.

### How secure is a Dedibox VPS?
Each Dedibox VPS is isolated from other virtual servers on the same host, ensuring security. However, best security practices, including regular updates, strong passwords, and firewall configurations, are essential to enhance security further.

### Are IP addresses for Dedibox VPS assigned statically?

> IP addresses for Dedibox VPS are generally assigned statically. However, in rare cases such as maintenance or replacement of a faulty host node, we may perform IP migrations, which can result in a change of IP address.

Reviewer: ```suggestion
IP addresses for Dedibox VPS are generally assigned statically. However, in rare cases, such as maintenance or replacement of a faulty host node, we may perform IP migrations, which can result in a change of IP address.
```

> You can find your updated IP address in the Dedibox console. Please note that we do not currently notify users of such changes. You can contact [technical support](https://console.online.net/en/assistance/ticket) if you experience any issues.

Reviewer: ```suggestion
You can find your updated IP address in the Dedibox console. Please note that we do not currently notify users of such changes. You can contact our [support team](https://console.online.net/en/assistance/ticket) if you experience any issues.
```

> You can find your updated IP address in the Dedibox console. Please note that we do not currently notify users of such changes. You may contact our [technical support](https://console.online.net/en/assistance/ticket) in case you experience any issues.

Reviewer 2: ```suggestion
You can find your updated IP address in the Dedibox console. Please note that we do not currently notify users of such changes. You can contact [technical support](https://console.online.net/en/assistance/ticket) if you experience any issues.
```
